project/publish-docs.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -e
+
+#########################################################################
+#
+# GUIDE TO USE OF THIS SCRIPT
+#
+#########################################################################
+#
+# - Set up npm scripts to perform the following acctions:
+#     - npm run build-docs
+#
+# - Alter the GITHUB_REPO to the appropriate URL
+#
+# - Setup GH_TOKEN and GH_REF if this is to be run on Travis
+#     - Create a new personal token on github here: https://github.com/settings/tokens/new
+#     - Run these two commands:
+#         - gem install travis
+#         - travis encrypt GH_TOKEN=<Github Token Here>
+#     - Copy the secure string into your .travis.yml file (as shown below)
+#     - Add GH_REF to /travis.yml as well:
+#     env:
+#         global:
+#         - secure: <Output from travis encrypt command>
+#         - GH_REF: github.com/<username>/<repo>.git
+#
+#########################################################################
+
+if [ "$BASH_VERSION" = '' ]; then
+ echo "    Please run this script via this command: './project/publish-docs.sh'"
+ exit 1;
+fi
+
+GITHUB_REPO="https://github.com/GoogleChrome/sw-testing-helpers/"
```

Comment:
I'd love to get it from git because it'll make the script reusable. At the moment I'm fighting an issue with travis :(